src/Pos/Util/BackupPhrase.hs

```diff
@@ -34,15 +34,17 @@ newtype BackupPhrase = BackupPhrase
 backupPhraseWordsNum :: Int
 backupPhraseWordsNum = 12
 
--- | Number of hashing rounds to get seed from phrase
-hashingRoundsNum :: Int
-hashingRoundsNum = 10000
+-- | Make backup phrase from list
+mkBackupPhrase12 :: [Text] -> BackupPhrase
```

Comment:
And I have to add one more commit to enable 9 word mnemonics in client-api (which does some checks)